commercetools-models/src/main/java/io/sphere/sdk/carts/commands/updateactions/SetLineItemPrice.java

```diff
@@ -0,0 +1,55 @@
+package io.sphere.sdk.carts.commands.updateactions;
+
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.carts.LineItem;
+import io.sphere.sdk.commands.UpdateActionImpl;
+import org.javamoney.moneta.Money;
+
+import javax.annotation.Nullable;
+
+/**
+ * Sets the price of a line item and changes the priceMode of the line item to ExternalPrice.
+ * If the price mode of the line item is ExternalPrice and no externalPrice is given, the external price is unset
+ * and the priceMode is set to Platform.
+ *
+ * {@doc.gen intro}
+ *
+ * {@include.example io.sphere.sdk.carts.commands.CartUpdateCommandIntegrationTest#setLineItemExternalPrice()}
+ */
+public final class SetLineItemPrice extends UpdateActionImpl<Cart> {
+
+    final private String lineItemId;
+    @Nullable
+    final private Money externalPrice;
```

Comment:
Looks like I forgot to mention this usage of `Money` in my last review, but we have to use `MonetaryAmount` in all of our API classes.